documentation/source/conf.py

```diff
@@ -303,8 +303,7 @@
 # If true, do not generate a @detailmenu in the "Top" node's menu.
 #texinfo_no_detailmenu = False
 
-# For now show the todos
-todo_include_todos = True
+#todo_include_todos = True
```

Comment:
```suggestion
todo_include_todos = False
```
this is still rendering

Reply:
Done.